services/brig/test/integration/API/User/Account.hs

```diff
@@ -141,9 +141,59 @@ tests _ at opts p b c ch g aws =
       testGroup
         "temporary customer extensions"
         [ test' aws p "domains blocked for registration" $ testDomainsBlockedForRegistration opts b
+        ],
+      testGroup
+        "re-send activation email"
+        [ test' aws p "put /users/:uid/email" $ testReSendActivationEmail b
         ]
     ]
 
+testReSendActivationEmail :: Brig -> Http ()
```

Comment:
It's not really *re*send any more, though, right?  That's just the use case we're doing this for now, but "send" and "re-send" are indistinguishable.  Do you think we should rename it?  Is there "re-" anywhere in the production code?  (If not I don't care all that much.)